### PR TITLE
Can render info dynamically, changed dummy data and schemas, changed logo, fixed bugs on carousel

### PR DESCRIPTION
**Pull request by enague**

### CODE REVIEW

File: client/src/components/App.jsx [hunk 1]
```diff
@@ -1,119 +1,115 @@
...
-		// axios
-		// 	.get("/locations/hostels/hosetelId/pictures")
-		// 	.then(response => {
-		// 		console.log(response.data[0]);
-		// 	})
-		// 	.catch(error => {
-		// 		console.log(error);
-		// 	});
+	componentDidMount() {
+		this.getHostelInfo();
```
Comment (datonedoe) [inline client/src/components/App.jsx:28]: i liked how you modulized ajax requests from componentDidMount

**Review by datonedoe — APPROVED** [review]
things look great here. Just wanna say you might want to include proptypes for each of React components. Dont your eslint tell you there were errors if you dont have proptypes? Because it should.

File: client/src/components/App.jsx (hunk 1 above)
Comment (enague) [reply client/src/components/App.jsx:28]: thank you! I tried hard to see where I could modularize my code. I even made a wifi, coffee, and modal component.

Comment (enague) [reply client/src/components/App.jsx:28]: No I'll talk to you more about proptypes. I don't know what that is

**Pull request merged**